### DISCUSSION TITLE
Can't security Onion url over linux

### DISCUSSION THREAD

**Discussion by MidLaunch** [Security-Onion-Solutions/securityonion#7995] [General]
**[errno 21 No such file or directory : 'nsm/wazuh/etc/ossec.conf'**
after trying command
`so-allow` ---> option a ----> enter ubuntu machine IP to allow firewall access----> errno 21 appears.

after sometime i run the command again and the result is different: ' already exist'.

Switch machine (both machine are running)

headover ubuntu and enter securityOnion IP url for access through **Analyst -80/tcp, 443/tcp**
---> **RESULT: Unable to connect**

I understand that the path is missing and a might be securityonion configuration I can do to fix it or maybe a particular download.

**Comment by dougburks** [comment]
If you didn't enable Wazuh during installation, then you probably wouldn't have `/nsm/wazuh/etc/ossec.conf`.

Is the firewall rule getting created? You can check for your analyst IP address like this:
```
sudo iptables -nvL | grep YOUR.ANALYST.IP.ADDRESS
```

**Reply by MidLaunch** [reply]
copy I will check on my way back from work.

**Comment by RagnarSTS** [comment]
A loooong time ago I was having login issues even though so-allow was completed; wound up being an ancient browser on the analyst machine.  I didn't spend the time to figure out why, just updating to a current version of chrome/firefox/edge/whatever fixed it right up.
